packages/matchbox/src/components/ProgressBar/ProgressBar.js

```diff
@@ -1,45 +1,51 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
-import styles from './ProgressBar.module.scss';
-
-class ProgressBar extends Component {
-  static displayName = 'ProgressBar';
-
-  static propTypes = {
-    /**
-     * Completion in percentage
-     */
-    completed: PropTypes.number.isRequired,
-
-    /**
-     * Bar color
-     */
-    color: PropTypes.oneOf(['orange', 'blue', 'navy', 'purple', 'red'])
-  };
-
-  static defaultProps = {
-    completed: 0,
-    color: 'orange'
-  }
-
-  render() {
-    const { completed = 0, color } = this.props;
-
-    let percentage = completed;
-
-    if (percentage > 100) {
-      percentage = 100;
-    } else if (percentage < 1) {
-      percentage = 0;
-    }
-
-    return (
-      <div className={classnames(styles.ProgressBar, styles[color])}>
-        <div className={styles.Progress} style={{ width: `${percentage}%` }}/>
-      </div>
-    );
-  }
+import { deprecate } from '../../helpers/propTypes';
+import styled from 'styled-components';
+import { margin, compose } from 'styled-system';
+import { createPropTypes } from '@styled-system/prop-types';
+import { Box } from '../Box';
+
+import { outerBase, innerBase, visualSize, calculatedWidth } from './styles.js';
+
+const system = compose(margin);
+
+export const StyledProgressBarOuter = styled(Box)`
+  ${outerBase}
+  ${visualSize}
+  ${system}
+`;
+
+export const StyledProgressBarInner = styled(Box)`
+  ${innerBase}
+  ${visualSize}
+  ${calculatedWidth}
+`;
+
+function ProgressBar(props) {
+  const { completed = 0, size } = props;
+
+  return (
+    <StyledProgressBarOuter as="div" size={size}>
+      <StyledProgressBarInner as="div" completed={completed} size={size} />
+    </StyledProgressBarOuter>
+  );
 }
 
+ProgressBar.displayName = 'ProgressBar';
+
+ProgressBar.propTypes = {
+  completed: PropTypes.number.isRequired,
+  color: deprecate(
+    PropTypes.oneOf(['orange', 'blue', 'navy', 'purple', 'red']),
+    'Color is always blue for now. This may be updated in the future.',
+  ),
+  size: PropTypes.oneOf(['normal', 'small']),
+  ...createPropTypes(margin.propNames),
+};
+
+ProgressBar.defaultProps = {
+  size: 'normal',
```

Comment:
I think default props changes still surface in Enzyme snapshots, so this would be a slightly-breaking change for the app. I think I would destructure `normal` to a default value in the component instead, like how `completed` is now

actually it might not matter at all since we're changing the color and completed default props too /shrug